__tests__/app.test.js

```diff
@@ -13,6 +13,40 @@ afterAll(() => {
   return db.end();
 });
 
+describe("Bad path", () => {
+  describe("Status 404: responds with 'Path does not exist' message, for: ", () => {
+    test("None existant path on /api/topics", () => {
+      return request(app)
+        .get("/api/topcs")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.message).toBe("Path does not exist");
+        });
+    });
+    test("None existant path on api/articles/:artcle_id", () => {
```

Comment:
Ideally we should move this error handler to the relevant describe block containing all tests for that one endpoint.